docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,108 @@
+
+You can scale resources based on the deployment you choose.
+* For Single server deployment, you can refer to the [official documentation](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#scale-resources) for scaling resources.
+* For Flexible server deployments, you can read more about scaling resources [here](https://docs.microsoft.com/en-gb/azure/mysql/flexible-server/concepts-compute-storage#scale-resources).
+
+### Monitoring Azure Database for MySQL
+
+By default, Azure Database for MySQL servers has monitoring enabled.
+You can monitor database metrics from the Azure portal.
+For more information, check the [official documentation](https://docs.microsoft.com/en-us/azure/mysql/concepts-monitoring).
+
+## Azure Database for PostgreSQL
+
+Azure Database for PostgreSQL is a fully managed PostgreSQL database provided by Azure. You can create a PostgreSQL database by selecting a database instance and storage according to your configurations. The IOPS is calculated based on the amount of storage you choose.
+In this section, you will create an Azure Database for PostgreSQL.
+
+### Recommendations
+
+* You should select Flexible or Single server deployment for Azure Database for PostgreSQL. You can refer to [official documentation](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to check and compare which option suits you best.
+* You should select a deployment that suits your IOPS requirements.
+    * For Single server deployments, IOPS is defined on the amount of storage provisioned(3 IOPS/GB).
+    * For Flexible server deployments, additional IOPS can be provisioned and the maximum threshold will be defined by the storage and the database instance chosen.
+
+### Steps
+
+For creating an Azure Database for PostgreSQL,
+* For Single server deployment, follow the [official documentation](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal).
+* For Flexible server deployment, follow this [official documentation](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/quickstart-create-server-portal).
+
+### Scale Performance
+
+**Scaling storage**:- You can enable the `storage auto grow` feature to automatically increase storage when it reaches a threshold without impacting the workload. Please note this feature is not yet available for Flexible server deployment for Azure Databases for PostgreSQL.
+
+You can scale resources based on the deployment you choose. 
+* For Single server deployment, you can refer to the [official documentation](https://docs.microsoft.com/en-gb/azure/postgresql/concepts-pricing-tiers#scale-resources) for scaling resources.
+* For Flexible server deployment, you can read more about scaling resources [here](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-compute-storage#scale-resources).
+
+### Monitoring Azure Database for PostgreSQL
```

Comment:
Ditto

Reply:
I have updated the monitor section to match the same structure as the cosmos DB section.